.gitignore

```diff
@@ -1,4 +1,6 @@
-release/*

```

Comment:
I see no reason to remove this from the gitignore. Especially because armake only works on Linux yet.